src/doc/en/developer/review.rst

```diff
@@ -134,7 +134,7 @@ the patches have been applied to a given release, the issue would
 remain in limbo.
 
 **No Patch Bombs**: Code that goes into Sage is peer-reviewed. If
-you show up with an 80,000 lines of code bundle that completely
+you show up with 80,000 lines of code bundle that completely
```

Comment:
I read the original as "with an 80,000-lines-of-code bundle", where "an" would be correct

Reply:
Oh, well I thought the article was dangling since it is not commonly used with numerals. If it was "eighty thousand" on the other hand, that would have been the case But maybe I am mistaken.